third_party/gtest-1.8.0/src/gtest.cc

```diff
@@ -3212,6 +3212,8 @@ void PrettyUnitTestResultPrinter::OnTestIterationEnd(const UnitTest& unit_test,
     PrintFailedTests(unit_test);
     printf("\n%2d FAILED %s\n", num_failures,
                         num_failures == 1 ? "TEST" : "TESTS");
+  } else {
+	  ColoredPrintf(COLOR_GREEN,  "[  ALL TESTS PASSED  ] \n");
```

Comment:
This is only required by Axxon, I don' think Axxon can check the return value of the test. This line can be removed when we get rid of Axxon in the future.